lte/gateway/c/oai/tasks/mme_app/mme_app_state_converter.cpp

```diff
@@ -236,14 +236,8 @@ void MmeNasStateConverter::ue_context_to_proto(
 {
   ue_ctxt_proto->Clear();
 
-  ue_ctxt_proto->set_imsi(ue_ctxt->imsi);
-  ue_ctxt_proto->set_imsi_len(ue_ctxt->imsi_len);
-  ue_ctxt_proto->set_member_present_mask(ue_ctxt->member_present_mask);
-
-  memcpy(
-    ue_ctxt_proto->mutable_imeisv(),
-    &ue_ctxt->imeisv.u.value,
-    IMEISV_BCD8_SIZE);
+  ue_ctxt_proto->set_imsi(ue_ctxt->emm_context._imsi64);
```

Comment:
The UE context proto message does not have any member field imsi or imsi_len any more. So lines 239 and 240 are not needed.